browser/net/url_context.cc

```diff
@@ -113,13 +113,13 @@ void BraveRequestInfo::FillCTXFromRequest(
   ctx->referrer = GURL(request->referrer());
   ctx->referrer_policy = request->referrer_policy();
 
-  auto* request_info = content::ResourceRequestInfo::ForRequest(request);
-  if (request_info) {
-    ctx->resource_type = request_info->GetResourceType();
-    if (auto* context = request_info->GetContext()) {
-      ctx->is_webtorrent_disabled = IsWebTorrentDisabled(context);
-    }
-  }
+  //auto* request_info = content::ResourceRequestInfo::ForRequest(request);
```

Comment:
We will deprecate this whole function, the only reason it exist - a bunch of tests that use it. So I think it is safe to temporarily comment it out (and link to https://github.com/brave/brave-browser/issues/6313)